UI/js-src/lsmb/ReconciliationLine.js

```diff
@@ -0,0 +1,32 @@
+define("lsmb/ReconciliationLine",
+       ["dojo/_base/declare",
+        "dojo/on",
+        "dijit/registry",
+        "dojo/dom-class",
+        "dijit/form/CheckBox"],
+       function(declare, on, registry, domClass, CheckBox) {
+           return declare("lsmb/ReconciliationLine", [CheckBox], {
+               publish: function(targetValue) {
```

Comment:
If you look at https://github.com/ehuelsmann/LedgerSMB/blob/master/UI/js-src/lsmb/PublishSelect.js#L10, you see that that function (also called `publish`) invokes an action on a channel. I'm missing that part below, right?